File: CS50x/Source Code/lesson6/agree.py
from cs50 import get_string

s = get_string("Do you agree?\n")

#if s == "Y" or s == "y":
if s.lower() in ["y", "yes"]:   #input YES or YeS is valid
    print("Agreed.")
elif s in ["N", "n", "No", "no"]:
    print("Not agreed.")

File: CS50x/Source Code/lesson1/agree.c
//logical operators

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //prompt user to agree.
    char c = get_char("Do you agree?\n");
    
    //check whether agreed.
    if (c == 'y' || c == 'Y')
    {
        printf("Agreed.\n");
    }
    else if (c == 'n' || c == 'N')
    {
        printf("Not agreed.\n");
    }
}